### PR TITLE
Ability to require search for a case list

### PR DESCRIPTION
**Pull request by orangejenny**
##### SUMMARY
https://dimagi-dev.atlassian.net/browse/USH-366

Depends on https://github.com/dimagi/formplayer/pull/775

##### RISK ASSESSMENT / QA PLAN
Will request QA.

##### PRODUCT DESCRIPTION
Adds an option to mark a specific menu's case list as requiring search text:
Screen Shot 2020-10-22 at 10 31 48 PM

Web Apps then won't display any cases until you search:
Screen Shot 2020-10-22 at 10 29 23 PM

### CODE REVIEW

File: corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/collections.js [hunk 1]
```diff
@@ -6,15 +6,17 @@ hqDefine("cloudcare/js/formplayer/menus/collections", function () {
...
-            'breadcrumbs',
-            'appVersion',
-            'appId',
             'persistentCaseTile',
+            'requireSearch',
+            'searchText',
             'tiles',
+            'title',
+            'type',
             'selections',
```
Comment (stephherbers) [inline corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/collections.js:20]: it took me way too long to realize you put these in alphabetical order

**Review by stephherbers — APPROVED** [review]
this will be a great feature

File: corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/collections.js (hunk 1 above)
Comment (orangejenny) [reply corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/collections.js:20]: ah, sorry, I thought about pulling that out as a separate commit and got lazy

**Review by snopoke — COMMENTED** [review]
Are there any formplayer changes for this?

Also will mobile just ignore this new attribute?

**Comment by orangejenny** [comment]
@snopoke Yeah, https://github.com/dimagi/formplayer/pull/775 and https://github.com/dimagi/commcare-core/pull/939 are both related to this, and mobile would ignore it.

But it's looking like the requirements for this feature are changing, so this is likely to get closed (along with the other two PRs).

**Comment by orangejenny** [comment]
Closing in favor of https://github.com/dimagi/commcare-hq/pull/28788

**Pull request closed without merging**